### ISSUE TITLE
Recover VOD still crashing

### ISSUE THREAD

**Issue by Sketchydoor** [TwitchRecover/TwitchRecover#130]
I'm using the old stream charts link. I paste in the link then press enter it searches for about 30 seconds then crashes? Anyone else have this?

**Comment by youngkiller1988** [comment]
yep me too if this program doesn't get updated soon there is no point in having it downloaded every feature has gone to shit
is the person even working on it any more there's been no update for 2 months

**Comment by Sketchydoor** [comment]
It worked for me yesterday and all of a sudden stopped working today again lol

**Comment by youngkiller1988** [comment]
yep same for me it cant find any vod link now

**Comment by TheYoungCormier** [comment]
The only reason the application is not working is because the websites that it scrapes from are changing. Developing a web scraping tool such as this... where it is specific information from a site requires consistent checking/updating. 

You can't blame a single developer for not having time to up keep an application he built for others for FREE and expect it to work 100% of the time.

I think you're expectations are too high for open source development especially for a web scraping functionality such as this one.

**Comment by youngkiller1988** [comment]
even when doing it manually it does not work
it would take two seconds to make a simple comment about whats going on but its been 2 months
if he plans on not continuing development of the program he should just say it its not that hard and i would understand

**Comment by TheYoungCormier** [comment]
you opened like 5 issues in the last 2 days.. complaining about the same issue.

**Comment by youngkiller1988** [comment]
yes and where is the developer i think this is a Abandoned program

**Comment by TheYoungCormier** [comment]
it might be if you spend 24 hours a day opening issues.. about the same problem. Are you looking to get every vod on twitch or something?

**Comment by youngkiller1988** [comment]
well since your here is this the right format for manual vod recovery 2021-06-17 12:59:16

**Comment by TheYoungCormier** [comment]
https://github.com/TwitchRecover/TwitchRecover/issues/96

**Comment by beaverfeaver12** [comment]
Well, it was fun while it lasted. This is a free program so I appreciate it even existing in this first place.

**Comment by nevereveneverreally** [comment]
daylamtayari updated the readme 2 weeks ago, so I wouldn't conclude it's abandoned. Give him a few days to respond and if he doesn't, his contact info is in the readme.

**Comment by profchimpington** [comment]
yeah i had my version work for a day, then the next, its not wanting to go past the loading the link then crashing.

**Comment by TheYoungCormier** [comment]
Found another work around.. 

Go the following and download ->  TwitchRecover-CLI-v1.2.exe

Use option 2 and input twitch tracker link... as long as the VOD was saved and is not deleted by twitch then the m3u8 will be generated.

**Comment by youngkiller1988** [comment]
hey did you block me i can't see your comments or links can you send me the link to my email

-------- Original Message --------

**Comment by profchimpington** [comment]
> yeah i had my version work for a day, then the next, its not wanting to go past the loading the link then crashing.

This issue seemed to resolve itself the day after i posted, the only remaining issue is the option 1 "get a live stream link" when i try that it crashes, i think this might have been what had happened when i tryed this out that had then caused the option 5 "recover VOD" issue.

**Comment by nevereveneverreally** [comment]
> This issue seemed to resolve itself the day after i posted, the only remaining issue is the option 1 "get a live stream link" when i try that it crashes, i think this might have been what had happened when i tryed this out that had then caused the option 5 "recover VOD" issue.

Is it still working for you? It hasn't worked at all for me since the issue opened.

**Comment by profchimpington** [comment]
> > This issue seemed to resolve itself the day after i posted, the only remaining issue is the option 1 "get a live stream link" when i try that it crashes, i think this might have been what had happened when i tryed this out that had then caused the option 5 "recover VOD" issue.
> 
> Is it still working for you? It hasn't worked at all for me since the issue opened.

yeah the VOD recovery is still working..... 
Like i said it just the "get live stream link" that seems to crash which is a different option then the VOD recovery option

**Comment by youngkiller1988** [comment]
> > > This issue seemed to resolve itself the day after i posted, the only remaining issue is the option 1 "get a live stream link" when i try that it crashes, i think this might have been what had happened when i tryed this out that had then caused the option 5 "recover VOD" issue.
> > 
> > 
> > Is it still working for you? It hasn't worked at all for me since the issue opened.
> 
> yeah the VOD recovery is still working.....
> Like i said it just the "get live stream link" that seems to crash which is a different option then the VOD recovery option

is vod recovery still working for you

**Comment by profchimpington** [comment]
> > > > This issue seemed to resolve itself the day after i posted, the only remaining issue is the option 1 "get a live stream link" when i try that it crashes, i think this might have been what had happened when i tryed this out that had then caused the option 5 "recover VOD" issue.
> > > 
> > > 
> > > Is it still working for you? It hasn't worked at all for me since the issue opened.
> > 
> > 
> > yeah the VOD recovery is still working.....
> > Like i said it just the "get live stream link" that seems to crash which is a different option then the VOD recovery option
> 
> is vod recovery still working for you

yesterday I did a windows update (not 11) and found the vod recovery was back to crashing trying to get the quality options. 
I tried last night uninstalling reinstalling the "setup" version, to no avail, I had also tried the no setup " .exe" version and was having the same prob. 
This evening I uninstalled, restarted, then reinstalled, thankfully it seems to be back to working order. 
Not sure if the windows update did anything, but certainly I had tried again before the uninstall, it was clearly doing the same thing on both versions, but the remove restart install seems to have it back to order. 
I'm not sure if its of any help for any of you, but this seems to have done the trick at least for me.

**Comment by youngkiller1988** [comment]
> > > > > This issue seemed to resolve itself the day after i posted, the only remaining issue is the option 1 "get a live stream link" when i try that it crashes, i think this might have been what had happened when i tryed this out that had then caused the option 5 "recover VOD" issue.
> > > > 
> > > > 
> > > > Is it still working for you? It hasn't worked at all for me since the issue opened.
> > > 
> > > 
> > > yeah the VOD recovery is still working.....
> > > Like i said it just the "get live stream link" that seems to crash which is a different option then the VOD recovery option
> > 
> > 
> > is vod recovery still working for you
> 
> yesterday I did a windows update (not 11) and found the vod recovery was back to crashing trying to get the quality options.
> I tried last night uninstalling reinstalling the "setup" version, to no avail, I had also tried the no setup " .exe" version and was having the same prob.
> This evening I uninstalled, restarted, then reinstalled, thankfully it seems to be back to working order.
> Not sure if the windows update did anything, but certainly I had tried again before the uninstall, it was clearly doing the same thing on both versions, but the remove restart install seems to have it back to order.
> I'm not sure if its of any help for any of you, but this seems to have done the trick at least for me.

sorry for all the questions but what method are you using links or manual

**Comment by profchimpington** [comment]
> > > > > > This issue seemed to resolve itself the day after i posted, the only remaining issue is the option 1 "get a live stream link" when i try that it crashes, i think this might have been what had happened when i tryed this out that had then caused the option 5 "recover VOD" issue.
> > > > > 
> > > > > 
> > > > > Is it still working for you? It hasn't worked at all for me since the issue opened.
> > > > 
> > > > 
> > > > yeah the VOD recovery is still working.....
> > > > Like i said it just the "get live stream link" that seems to crash which is a different option then the VOD recovery option
> > > 
> > > 
> > > is vod recovery still working for you
> > 
> > 
> > yesterday I did a windows update (not 11) and found the vod recovery was back to crashing trying to get the quality options.
> > I tried last night uninstalling reinstalling the "setup" version, to no avail, I had also tried the no setup " .exe" version and was having the same prob.
> > This evening I uninstalled, restarted, then reinstalled, thankfully it seems to be back to working order.
> > Not sure if the windows update did anything, but certainly I had tried again before the uninstall, it was clearly doing the same thing on both versions, but the remove restart install seems to have it back to order.
> > I'm not sure if its of any help for any of you, but this seems to have done the trick at least for me.
> 
> sorry for all the questions but what method are you using links or manual

did you mean for getting the vod info for the option 5 "Vod recover"? 
I have been using the twitchtracker and c/v the link into the TwitchRecover via option 5 then option 1 when asked for, so I guess link. 
I have never tried the second option to do it manual.

**Comment by ganzgeschmeidig** [comment]
same here.. vod recover is out of order ;< the app crashes :<

**Comment by profchimpington** [comment]
> same here.. vod recover is out of order ;< the app crashes :<

I occasionally find it crashes, if it does I restart the pc, it might take a couple of goes but it will usually end up working again, not sure what it is that is causing the problem, but the restart seems to help. 
Its now part of my start up routine, start pc, check the vod recovery app, restart if required and retest.
